<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createOwner</class>
 <widget class="QMainWindow" name="createOwner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FEFEFE;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1440</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    border: none; 
}

QFrame#header {
    border-bottom: 1px solid #DEDEDE;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="logo" native="true">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>21</y>
       <width>244</width>
       <height>48</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>3</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Acrom</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Артемида</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>2</y>
        <width>40</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Иконка2.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>15</y>
       <width>247</width>
       <height>57</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Onest</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 16px 24px;
color: #242424;
border: 2px solid #DEDEDE;
border-radius: 16px;</string>
     </property>
     <property name="text">
      <string>Назад к карте питомца</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>15</y>
       <width>211</width>
       <height>57</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Onest</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 16px 24px;
color: #242424;
border: 2px solid #DEDEDE;
border-radius: 16px;</string>
     </property>
     <property name="text">
      <string>Закрыть программу</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>125</y>
      <width>269</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onest Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 12px 24px;
color: white;
border: none;
border-radius: 16px;
background-color: #174EB9;</string>
    </property>
    <property name="text">
     <string>Сохранить изменения</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>123</y>
      <width>292</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onest</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 12px 24px;
color: #242424;
border: none;</string>
    </property>
    <property name="text">
     <string>Закрыть редактирование</string>
    </property>
   </widget>
   <widget class="QFrame" name="info_owner">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>224</y>
      <width>644</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    border: none; 
}

QFrame#info_owner {
    border: 1px solid #DEDEDE;
	border-radius: 16px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>21</y>
       <width>596</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Onest SemiBold</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #242424;</string>
     </property>
     <property name="text">
      <string>Информация о владельце</string>
     </property>
    </widget>
    <widget class="QWidget" name="aboutpet" native="true">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>70</y>
       <width>596</width>
       <height>264</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>396</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #DEDEDE;
border-radius: 6px;
padding: 6px 16px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите фамилию</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>3</y>
        <width>168</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #383838;</string>
      </property>
      <property name="text">
       <string>Фамилия</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>52</y>
        <width>396</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #DEDEDE;
border-radius: 6px;
padding: 6px 16px;
</string>
      </property>
      <property name="placeholderText">
       <string>Введите имя</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>55</y>
        <width>168</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #383838;</string>
      </property>
      <property name="text">
       <string>Имя</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>107</y>
        <width>168</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #383838;</string>
      </property>
      <property name="text">
       <string>Отчество</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>104</y>
        <width>396</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #DEDEDE;
border-radius: 6px;
padding: 6px 16px;
</string>
      </property>
      <property name="placeholderText">
       <string>Введите отчество</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>159</y>
        <width>168</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #383838;</string>
      </property>
      <property name="text">
       <string>Номер телефона</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>156</y>
        <width>396</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #DEDEDE;
border-radius: 6px;
padding: 6px 16px;
</string>
      </property>
      <property name="placeholderText">
       <string>Укажите номер телефона</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>212</y>
        <width>396</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #DEDEDE;
border-radius: 6px;
padding: 6px 16px;
</string>
      </property>
      <property name="placeholderText">
       <string>Укажите электронную почту</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>202</y>
        <width>168</width>
        <height>56</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onest</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #383838;</string>
      </property>
      <property name="text">
       <string>Электронная почта</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
